tests/integration/sdk_api/alerts/test_alerts.py

```diff
@@ -692,9 +692,6 @@ def _create_alerts_test(self, project_name, alert1, alert2):
                     "severity": "warning",
                     "when": ["now"],
                     "condition": "failed",
-                    "secret_params": {
-                        "webhook": "https://hooks.slack.com/services/",
-                    },
```

Comment:
change the notification name to `git` too while you're at it (instead of `slack_jobs`)